content/docs/command-reference/check-ignore.md

```diff
@@ -37,6 +37,12 @@ ones that are ignored indeed are printed back.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+- `--stdin` - read paths from standard input instead of providing `targets`.
+  Useful for interactive debugging (exit with `Ctrl+C`) and POSIX pipes.
```

Comment:
> exit with `Ctrl+C`

Is that correct @karajan1001 ? Thanks

Reply:
I see it works, although it prints an error... If there's a better way please update the doc.

![image](https://user-images.githubusercontent.com/1477535/89701682-cfee1180-d8fe-11ea-8b99-d6ae6e555ca2.png)